recipe/patch_yaml/parcels.yaml

```diff
@@ -8,3 +8,16 @@ then:
   - replace_depends:
       old: zarr >=2.11.0
       new: zarr >=2.11.0,!=2.18.0
+
+---
+
+# https://github.com/conda-forge/parcels-feedstock/pull/730
```

Comment:
This is a 404.

Reply:
Picked the wrong issue number. Fixed now